intezer_sdk/api.py

```diff
@@ -219,6 +219,12 @@ def assert_any_on_premise(self):
         if self.on_premise_version:
             raise errors.UnsupportedOnPremiseVersionError('This endpoint is not available yet on on-premise')
 
+    @staticmethod
+    def is_intezer_site_available() -> bool:
+        response = requests.get('https://analyze.intezer.com/api/v2-0/is-available')
+        is_available = response.json().get('result', {}).get('Is available')
```

Comment:
I think json will fail if the response didn't returned json formatted, I suggest we check the status code before